src/platform/plugins/shared/data_view_editor/public/components/flyout_panels/flyout_panel.tsx

```diff
@@ -117,10 +117,14 @@ export const useFlyoutPanelContext = (): Context => {
 };
 
 const componentStyles = {
-  flyoutColumn: css({
-    height: '100%',
-    overflow: 'hidden',
-  }),
+  flyoutColumn: () =>
+    css({
+      height: '100%',
+      overflow: 'hidden',
+      [useEuiMaxBreakpoint('m')]: {
```

Comment:
As `componentStyles` is defined outside of React component I'm afraid `useEuiMaxBreakpoint` won't work here :(

AFAIK we should use `euiMaxBreakpoint` and pass theme (example: https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-react-field/src/field_button/field_button.tsx#L176 - we do the same for font size here, I can't find breakpoint example, but the flow is similar).

I'll test locally tomorrow, but in case you need any guidance with this pattern let me know! It's not super obvious how to use it instead of hooks

Reply:
Alternatively we can just use hook inside of react component and pass our current styles + mobile styles in array for `css` prop, but I'd avoid mixing two different syntaxes. Let's use it as a last resort, if the above wouldn't work